RadioItem/RadioItem.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Provides a MyTheme Item component and interactive radio toggle icon.
+ *
+ * @example
+ * <RadioItem>Item</RadioItem>
+ *
+ * @module my-theme/RadioItem
+ * @exports RadioItem
+ * @exports RadioItemBase
+ */
+
+import kind from '@enact/core/kind';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ToggleIcon from '../ToggleIcon';
+import ToggleItem from '../ToggleItem';
+
+import componentCss from './RadioItem.module.less';
+
+/**
+ * Renders an `Item` with a radio-dot icon.
+ *
+ * @class RadioItem
+ * @memberof my-theme/RadioItem
+ * @extends my-theme/ToggleItem.ToggleItem
+ * @omit iconComponent
+ * @ui
+ * @public
+ */
+const RadioItemBase = kind({
+	name: 'RadioItem',
+
+	propTypes: /** @lends my-theme/RadioItem.RadioItem.prototype */ {
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal Elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * * `radioItem` - The root class name
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		css: PropTypes.object
+	},
+
+	styles: {
+		css: componentCss,
+		className: 'radioItem',
+		publicClassNames: true
+	},
+
+	render: (props) => (
+		<ToggleItem
+			data-webos-voice-intent="SelectRadioItem"
```

Comment:
These `data-webos-*` props are not documented so I'm hesitant to include them. On one hand, they're useful for webOS themes but I'm not even clear on how they work so it might be best to exclude them. Thoughts?

Reply:
I think I'm indifferent toward these. On the one hand, it does at least hint at a feature of webOS that someone may be interested in looking into further, and accessibility is great. On the other, if it's not documented, and its purpose isn't explicitly clear, it appears as cruft, which may be copy/pasted into places it really shouldn't belong.

Reply:
I would lean away from including any undocumented or private APIs since it promotes adoption that might inhibit our evolution of those features.

Reply:
Sounds reasonable. I'll remove them.